lib/config/migrations/migrations-service.ts

```diff
@@ -79,6 +80,7 @@ export class MigrationsService {
     IgnoreNodeModulesMigration,
     PackageNameMigration,
     PackagePatternMigration,
+    PackagesMigration,
```

Comment:
We need to makes sure the order of migrations works as expected. We've some properties which changed multiple times. Those need to be migrated in order.

Reply:
If we retain existing tests, it hopefully covers them fairly well

Reply:
Order of properties will be saved, because property renaming happens only in one specific migration, another migrations can only change value of property

Reply:
What I mean is, that we renamed `packageRules` multiple times, so migration need to be in order to have the final right property name.